core/src/test/resources/test_mutation_effect_summaries.tsv

```diff
@@ -30,7 +30,7 @@ BRCA1	T100*	Likely Loss-of-function	The mutation effect description for truncati
 ### Truncating mutations in TSG that not curated
 PPP2R2A	T100*	Unknown
 ### Truncating mutation in both
-GATA3	T100*	Likely Loss-of-function	The mutation effect description for truncating mutations in GATA3 is: Truncating mutations in the GATA3 gene result in loss of GATA3 protein expression which while sufficient to enable tumor dedifferentiation and dissemination, is not associated with tumor development. This data indicates that GATA3 may not be a bona fide tumor suppressor, but rather comprises a crucial differentiation factor in cancer development (PMID: 18242514). Expression of truncating GATA3 mutations in breast cancer cell lines led to increased protein stability and impaired interaction with chromatin and DNA binding (PMID: 24758297, 26637396). Additionally, GATA3 truncating mutations in breast cancer resulted in reduced GATA3 transactivating activity with abnormalities in cellular invasiveness (PMID: 23426732), and displayed sensitivity to inhibitors of the histone methyltransferases G6A/GLP, upregulated in many cancer types (PMID: 27588951) .
+GATA3	T100*	Likely Loss-of-function	The mutation effect description for N-terminal truncating mutations in GATA3 is: The GATA3 1_307trunc mutations occur in the N-terminal region of GATA3 and result in the loss of the zinc finger 2 domain and the C-terminus. GATA3 truncating mutations in this range have been identified in breast cancer (PMID: 27251275). Truncated N-terminal transcripts of GATA3 have been identified as inactivating as measured by reduced transcriptional regulation of genes compared to wildtype and truncated C-terminal transcripts of GATA3 (PMID: 27588951). Truncated N-terminal transcripts of GATA3 are suggested to negatively impact the dimerization of the protein (PMID: 27588951, 31107186, 25410484).
```

Comment:
We should not use `1_307trunc mutations` here, but name of the string. I can fix later.